storage/conformance_test.go

```diff
@@ -172,7 +174,24 @@ func TestSigningV4Conformance(t *testing.T) {
 					style = BucketBoundHostname(tc.BucketBoundHostname)
 				}
 
-				gotURL, err := SignedURL(tc.Bucket, tc.Object, &SignedURLOptions{
+				if tc.EmulatorHostname != "" {
+					t.Setenv("STORAGE_EMULATOR_HOST", tc.EmulatorHostname)
```

Comment:
Nice use of t.Setenv; wondering if we should be using it elsewhere since it's a relatively recent addition.

Reply:
Yeah, there are likely various areas it could be used. I remember trying to use it a while back but we were still supporting previous versions.